CISS/transforms/CISS_Patient_Value_Map.py

```diff
@@ -0,0 +1,115 @@
+import Global_Constants
+
+ciss_patient_col_specific_value_maps = {
+    'AGE': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'0': 'Less than one year old', '999': 'Unknown'}
+        }
+    ],
+    'SEX': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'3': 'Female, pregnant - 1st trimester (1st-3rd month)',
+                        '4': 'Female, pregnant - 2nd trimester (4th-6th month)',
+                        '5': 'Female, pregnant - 3rd trimester (7th-9th month)',
+                        '6': 'Female, pregnant - trimester unknown'} | Global_Constants.sex_shared_mapping
+        }
+    ],
+    'HEIGHT': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'999': 'Unknown'}
+        }
+    ],
+    'WEIGHT': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': Global_Constants.weight_shared_mapping
+        }
+    ],
+    'ROLE': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': Global_Constants.role_shared_mapping
+        }
+    ],
+    'SEATLOC': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'11': 'Front Left', '12': 'Front middle', '13': 'Front right', '14': 'Front other',
+                        '21': 'Second left', '22': 'Second middle', '23': 'Second right', '24': 'Second other',
+                        '31': 'Third left', '32': 'Third middle', '33': 'Third right', '34': 'Third other',
+                        '41': 'Fourth left', '42': 'Fourth middle', '43': 'Fourth right', '44': 'Fourth other',
+                        '51': 'Fifth left', '52': 'Fifth middle', '53': 'Fifth right', '54': 'Fifth other',
+                        '97': 'In or on unenclosed area', '98': 'Other enclosed area', '99': 'Unknown'}
+        }
+    ],
+    'ENTRAP': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'2': 'Could not exit vehicle due to jammed doors',
+                        '3': 'Could not exit vehicle due to external circumstances (specify)'
+                        } | Global_Constants.entrapped_shared_mapping
+        }
+    ],
+    'EJECTTYPE': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': Global_Constants.ejection_shared_mapping
+        }
+    ],
+    'TREATMENT': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'1': 'Treatment at scene - non-transported', '2': 'Transported and released',
+                        '4': 'Dead on Arrival (DOA) at hospital', '5': 'Dead Prior To Admission',
```

Comment:
Also nitpicky, but if we want to match case, we can make sure to do so here as well: 'Dead prior to arrival" instead of 'Dead Prior To Arrival'

Reply:
Yep good call, updated the casing here